packages/dialtone-tokens/themes/config.js

```diff
@@ -39,7 +164,11 @@ function _setStyleTag (id, content, rootNode) {
       rootNode?.appendChild(style);
     }
   } else {
-    rootNode.querySelector('#' + id).innerHTML = content;
+    const existingTag = rootNode.querySelector('#' + id);
+    // Only update if content changed (performance optimization)
+    if (existingTag.innerHTML !== content) {
+      existingTag.innerHTML = content;
+    }
```

Comment:
I think this is actually hurting performance rather than helping it. Doing a content comparison is likely a much heavier operation than just directly setting the content.